### PR TITLE
Reduce cardinality GRPC peer remove from metrics

### PR DESCRIPTION
**Pull request by cross311**
Peer is creating a bunch of cardinality aspects in metrics.  More of a logging thing rather then a metrics.

### CODE REVIEW

**Comment by plameniv** [comment]
Bump the patch version in package.json

**Comment by plameniv** [comment]
Should remove the `peer: call.getPeer()` from the 2 `metrics.increment` calls as well?

File: packages/grpc/src/internal/GrpcInputSource.ts [hunk 1]
```diff
@@ -137,7 +137,6 @@ export class GrpcInputSource implements IInputSource, IRequireInitialization {
 
                     this.metrics.increment(GrpcMetrics.RequestReceived, {
                         path: method.path,
-                        peer: call.getPeer(),
```
Comment (michaelliao5) [inline packages/grpc/src/internal/GrpcInputSource.ts:140]: what does getPeer do?

**Review by michaelliao5 — APPROVED** [review]

File: packages/grpc/src/internal/GrpcInputSource.ts (hunk 1 above)
Comment (plameniv) [reply packages/grpc/src/internal/GrpcInputSource.ts:140]: `Get the endpoint this call/stream is connected to. Returns the URI of the endpoint`

**Pull request merged**